src/sdk/resources/auto_detect_compilers.xrc

```diff
@@ -3,12 +3,12 @@
 	<object class="wxScrollingDialog" name="dlgAutoDetectCompilers">
 		<title>Compilers auto-detection</title>
 		<centered>1</centered>
-		<style>wxDEFAULT_DIALOG_STYLE|wxRESIZE_BORDER|wxMAXIMIZE_BOX</style>
+		<style>wxCAPTION|wxRESIZE_BORDER|wxMAXIMIZE_BOX</style>
 		<object class="wxBoxSizer">
 			<orient>wxVERTICAL</orient>
 			<object class="sizeritem">
 				<object class="wxStaticText" name="ID_STATICTEXT2">
-					<label>Note: After auto-detection, at least one compiler&apos;s master path is still empty and therefore invalid.&#x0A;Inspect the list below and change the compiler&apos;s master path later in the compiler options.&#x0A;Select you favourite default compiler here:</label>
+					<label>Note: After auto-detection, either the default compiler was not detected or at least none compiler&apos;s master&#x0A;path is still empty and therefore invalid. Inspect the list below and select another default compiler if it &#x0A;was not found or and change the compiler&apos;s master path later in the compiler options if the compiler is invalid.&#x0A;Select you favourite default compiler here:</label>
```

Comment:
`found or and change` - what is the meaning of this?

Reply:
Looks like I did not save the update I did....

Reply:
The code change to resolve this have been pushed.